### PR TITLE
Use a single manifest file for the wasm api tests.

### PR DESCRIPTION
**Pull request by mykmartin**
This changes the `arcs_kt_schema` rule to only accept a single input
manifest (which is the same as `arcs_cc_schema`). There are no current
uses of arcs_kt_schema with more than one manifest.

### CODE REVIEW

**Comment by csilvestrini** [comment]
> This changes the `arcs_kt_schema` rule to only accept a single input
> manifest (which is the same as `arcs_cc_schema`). There are no current
> uses of arcs_kt_schema with more than one manifest.

I think this is something we actually want to support in the future though. Clients may have a bunch of schema files that they want to generate code from, all in one go.

**Comment by mykmartin** [comment]
> I think this is something we actually want to support in the future though. Clients may have a bunch of schema files that they want to generate code from, all in one go.

The problem is that I don't know how to turn a src reference like `//src/dir:filegroup` into the actual file(s) contained therein, inside the build rule code. We need the input filename to generate the output .kt filename. I added the `out` parameter to the rule to solve this for the single input case (which is what C++ currently does).

I'd be happy to have multiple src targets (for both languages), but couldn't figure out how to. The bazel docs weren't much help here.

**Comment by csilvestrini** [comment]
Can you just do something similar to what we do for kotlin?

```
    for src in srcs:
        out = _output_name(src, "_GeneratedSchemas.kt")
```

And instead of using a filegroup, use a glob or list the files one by one.

**Comment by mykmartin** [comment]
No, that's the thing that doesn't work. Now that the manifest file lives in a different directory, and IIUC needs to be accessed via a filegroup target, `src` is '//src/wasm/tests:manifest-srcs' instead of 'schemas.arcs'. But we need a valid filename to generate the .kt file from.

Can I use a glob to read files from a different dir - specifically one that is *not* under the current dir?

**Comment by mykmartin** [comment]
Updated to a much better mechanism for accessing the manifest file which means we can keep the multi-src input for kt schema builds. Thanks for the tip Cameron.

File: src/wasm/tests/BUILD [hunk 1]
```diff
@@ -1,12 +1,18 @@
 load("//third_party/java/arcs/build_defs:build_defs.bzl", "arcs_ts_test")
 
+exports_files(["manifest.arcs"])
+
+filegroup(
+    name = "manifest_srcs",
+    srcs = ["manifest.arcs"],
+)
```
Comment (csilvestrini) [inline src/wasm/tests/BUILD]: I don't think you need this filegroup at all anymore. AFAIK, you should be able to just add "manifest.arcs" as a dep below.

File: src/wasm/tests/wasm-api-test.ts [hunk 2]
```diff
@@ -22,7 +22,17 @@ import '../../services/random-service.js';
 
 
 class TestLoader extends Loader {
+  constructor(readonly env: string) {
+    super();
+console.log(env);
```
Comment (csilvestrini) [inline src/wasm/tests/wasm-api-test.ts]: delete? 
...or indent

Comment (mykmartin) [reply src/wasm/tests/wasm-api-test.ts]: Deleted (leftover debugging)

File: src/wasm/tests/BUILD (hunk 1 above)
Comment (mykmartin) [reply src/wasm/tests/BUILD]: Tried that; didn't work:
```
ERROR: /usr/local/google/home/mykmartin/code/arcs/src/wasm/tests/BUILD:10:1: in deps attribute of run_in_repo_test rule //src/wasm/tests:wasm-api-test: source file '//src/wasm/tests:manifest.arcs' is misplaced here (expected no files).
```

I also tried adding it as a `data` field, no dice.

**Pull request merged**